clients/fossid-webapp/src/main/kotlin/Extensions.kt

```diff
@@ -670,7 +670,7 @@ suspend fun FossIdRestService.extractArchives(
 
```

Comment:
Commit message nit: I think, some commas are missing
* before "and the function..."
* before "so they are not visible"
* before "and their licenses"
* maybe before "as it is deleted..."